### ISSUE TITLE
Multiple kernels with different output sizes causing corrupt output

### ISSUE THREAD

**Issue by sroboubi** [gpujs/gpu.js#293]
```js
        var input = new Uint16Array(20);
        for (var i = 0; i < input.length; i++) {input[i] = 2;}
        var gpu = new GPU();

        var load = gpu.createKernel(function(input)
        {
            return input[this.thread.x];
        }).setOutput([input.length]).setOutputToTexture(true);

        var cf = gpu.createKernel(function()
        {
            return 5;
        }).setOutput([5]);

        console.log(input);
        var tex1 = load(input);
        console.log(tex1);
        console.log(tex1.toArray(gpu));
        var output = cf();
        console.log(output);
```
The output of this should be [5, 5, 5, 5, 5]. Instead I get [5, 5, 0, 5, 5]. If I change the size of the input array to something larger I get more zeroes in my output. It seems like the size of the input array should make no difference. The cf() function appears to be completely independent of the load(), so why does the size of the output of one affect the other?

Also, could someone please let me know if this project is still active? I would very much like to use this library but I'm concerned about lack of documentation/examples and possible bugs. If I have to find another solution I would rather know as soon as possible. Thanks.

**Comment by robertleeplummerjr** [comment]
Project is still active.  What video card are you using?

**Comment by robertleeplummerjr** [comment]
There is a chance that it could be.  Could you test on those machines and let me know if you see a difference?

**Comment by robertleeplummerjr** [comment]
Something we've been tossing around is the idea of detecting what system you are on and simply use a driver for that system, this could be a catalyst.

**Comment by sroboubi** [comment]
Sure, I'll let you know the results in a few hours. I'll need to do this test at home with my personal machines. Note that the code you see is the full thing, you can copy and paste that into a blank html page and if this is actually a bug I would expect you would see the same strange results that I see.

**Comment by sroboubi** [comment]
I just tried this with an nvidia 840M and another Intel HD Graphics 520 with an updated driver, and the test worked fine in both cases. I suspect my video card and/or driver are at fault. Thanks for pointing me in the right direction.

**Comment by robertleeplummerjr** [comment]
Lets leave this open for future testing if at all possible.  We do want to resolve this so that all major cards simply run.

**Comment by sroboubi** [comment]
OK. In that case here is the info for my system:
Intel HD Graphics 520
The problem happened with driver version 21.19.xx (don't remember the exact version)
The problem was resolved with driver version 21.20.16.4574

**Comment by robertleeplummerjr** [comment]
Ok, so they resolved it.  Cool.  Ty for this information, it'll likely help someone in the future.
